File: Recursion & Backtracking/TowerOfHanoi.py
def ToH(n, From, To, Mid):

    if(n==0):
        return;

    ToH(n-1, From, Mid, To)

    print("Disc", n, "From Rod", From, "To Rod", To)

    ToH(n-1, Mid, To, From)

ToH(3, "A", "C", "B")

File: 6. Recursion & Backtracking/TowerOfHanoi.cpp
#include <iostream>
#include <conio.h>
#include <stdio.h>

void ToH(int n, char from, char to, char mid)
{
    if(n==0)
    {
        return;
    }

    ToH(n-1, from, mid, to);
    std::cout<<"Disc "<<n<<" from rod "<<from<<" to rod "<<to<<std::endl;
    ToH(n-1, mid, to, from);
}

int main()
{
    system("cls");

    int n=3;

    ToH(n, 'A', 'C', 'B');

    return 0;
}
